ipi/engine/motion/dynamics.py

```diff
@@ -55,6 +56,8 @@ def __init__(
         fixcom=False,
         fixatoms=None,
         nmts=None,
+        efield=None,
+        bec=None,
     ):
```

Comment:
is there a reason to have two separate objects for this? seems it'd be appropriate at the very least to condense them in a single eda object

Reply:
I think these are leftovers? Please clean up this file from all the comments. You may want to just copy it over from master if you have now moved everything into the driven_dynamics.py